ui-tests/cypress/support/pageObjects/users/profileTypes/ProfileTypesPage.js

```diff
@@ -1,5 +1,76 @@
-import {DATA_TESTID, htmlElements, WebElement} from "../../WebElement.js";
+import { htmlElements } from '../../WebElement.js';
 
-import Content from "../../app/Content.js";
+import Content   from '../../app/Content.js';
+import AppPage      from '../../app/AppPage.js';
 
-export default class ProfileTypesPage extends Content {}
\ No newline at end of file
+import AddPage  from './AddPage.js';
+import EditPage from './EditPage.js';
+
+export default class ProfileTypesPage extends Content {
+
+  addButton = `${htmlElements.button}.ProfileType__add`;
+  editAction = `${htmlElements.li}.ProfileTypeListMenuAction__menu-item-edit`;
+  deleteAction = `${htmlElements.li}.ProfileTypeListMenuAction__menu-item-delete`;
+
+  getTable() {
+    return this.getContents()
+               .find(htmlElements.table);
+  }
+
+  getTableRows() {
+    return this.getTable()
+               .children(htmlElements.tbody)
+               .children(htmlElements.tr);
+  }
+
+  getTableRow(code) {
+    return this.getTable()
+               .find(`#${code}-actions`)
+               .closest(htmlElements.tr);
+  }
+
+  getTableRowActions(code) {
```

Comment:
Please, create a 'KebabMenu' class to handle the the different actions as it is for other page objects